nautilus_trader/adapters/binance/futures/data.py

```diff
@@ -157,3 +158,11 @@ def _handle_mark_price(self, raw: bytes) -> None:
         )
         generic = CustomData(data_type=data_type, data=data)
         self._handle_data(generic)
```

Comment:
Good call leaving this in, so we don't immediately break by changing to the first class `MarkPriceUpdate`.
I'll make sure to deprecate this one in the release notes for a cycle.